scripts/protocgen.sh

```diff
@@ -4,6 +4,9 @@
 # docker build --pull --rm -f "contrib/devtools/Dockerfile" -t cosmossdk-proto:latest "contrib/devtools"
 # docker run --rm -v $(pwd):/workspace --workdir /workspace cosmossdk-proto sh ./scripts/protocgen.sh
 
+echo "Formatting protobuf files"
```

Comment:
why the move up higher?

Reply:
It made in v0.50 two `make proto-gen` after each other to behave differently (as the original proto files may have changed after the generation).

See commit history of https://github.com/cosmos/cosmos-sdk/pull/17447